experimental/generation/generator/src/library/mergeAssets.ts

```diff
@@ -703,6 +703,20 @@ async function mergeDialogs(schemaName: string, oldPath: string, newPath: string
     await copySingleFile(newPath, mergedPath, schemaName + '.' + locale + '.lu.dialog', feedback)
 }
 
+/**
+ * @description: Get the trigger name
+ * @param trigger trigger from main.dialog file
+ */
+function getTriggerName(trigger: any): string {
+    let triggerName: string
+    if (typeof trigger !== 'string') {
+        triggerName = trigger['$id']
```

Comment:
>$id [](start = 31, length = 3)

Please change this to 'id'.  I'm in the process of removing $id.

Reply:
OK.